src/pages/my-resources/MyResources.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+
+import Tab from '~/components/tab/Tab'
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+
+import { tabsData } from '~/pages/my-resources/MyResources.constants'
+import { styles } from '~/pages/my-resources/MyResources.styles'
+
+const MyResources = () => {
+  const [activeTab, setActiveTab] = useState<string>('')
+  const { t } = useTranslation()
+
+  const handleClick = (tab: string) => {
+    setActiveTab(tab === activeTab ? '' : tab)
```

Comment:
I guess, it will be better to: 
tab === activeTab && setActiveTab(tab)

Reply:
It was needless, i changed it to setActiveTab(tab)